### ISSUE TITLE
In the last release the updated build released example projects to central and bintray

### ISSUE THREAD

**Issue by graemerocher** [micronaut-projects/micronaut-graphql#104]
The build for this project is including example projects in the release process.

See https://bintray.com/micronaut/core-releases-local/graphql-example-todo-spqr

Seems to have happened after updating the build

/cc @alvarosanchez @ilopmar

**Comment by alvarosanchez** [comment]
This should have prevented it:

https://github.com/micronaut-projects/micronaut-graphql/blob/master/build.gradle#L18-L20

**Comment by ilopmar** [comment]
It's not working. I've run `./gradlew pTML` and

![image](https://user-images.githubusercontent.com/559192/91164979-3bdac300-e6d0-11ea-9eee-a361a9bdaa28.png)

**Comment by ilopmar** [comment]
Modifying that `if` to use `contains` make it works

```
if (subproject.parent.name == "examples" || subproject.name.contains("example")) {
```

@alvarosanchez if you are ok with this I'll submit a PR to the build project

**Comment by alvarosanchez** [comment]
> @alvarosanchez if you are ok with this I'll submit a PR to the build project

You mean to change the build plugins to exclude example projects globally? If so, then yes, go ahead.

**Comment by ilopmar** [comment]
No, I was taking about this https://github.com/micronaut-projects/micronaut-project-template/pull/41

**Comment by alvarosanchez** [comment]
Then it's not in the build project.

Changing the template is fine, but it only applies to newly created repos from now on.

For existing projects... maybe a condition could be introduced [here](https://github.com/micronaut-projects/micronaut-build/blob/master/src/main/groovy/io/micronaut/build/MicronautPublishingPlugin.groovy#L24) so that we can remove those lines from all builds.

**Comment by ilopmar** [comment]
Submitted PR to fix it in the build project: https://github.com/micronaut-projects/micronaut-build/pull/38

**Comment by ilopmar** [comment]
Fixed by #106
